### DISCUSSION TITLE
Use flutter_wechat_assets_picker for Image Gallery selector with external urls, slideshow and web

### DISCUSSION THREAD

**Discussion by heralight** [fluttercandies/flutter_wechat_assets_picker#486] [Q&A]
Hi everybody!

Thank you for this great project!

My goal is to have something equivalent to the picker's image gallery, but whose content would come from network URLs and be passed as input to the picker, e.g. a list of URLs.
Finally, this version should be Web-compatible.

In the custom examples section, I saw the DirectoryFileAssetPicker example. Is it possible to create a variant of this implementation so that it takes a list of image or video URLs and creates AssetEntity and then passes them to the picker? And of course this version must be Web-compatible.

Another question: is it possible to have a dynamic option to activate / deactivate the selection, to have a slideshow mode for example.

Finally, in the examples, I couldn't find a way of simply specifying the path of a particular media directory. Do you have a suggestion, please?

Thank you in advance,

Alexandre

**Comment by AlexV525** [comment]
Sounds like you can build up a new picker on your own, regardless of the foundation of this plugin.
